src/tests/test_utils.py

```diff
@@ -26,3 +30,43 @@ def get_every_minute():
         list_of_times.append(time_minutes)
         minutes += 1
     return list_of_times
+
+
+def test_datetime_must_have_timezone():
+    """Test function datetime_must_have_timezone"""
+
+    time_now = datetime.now(timezone.utc)
+
+    # check functions works
+    datetime_must_have_timezone(None, time_now)
+
+    with pytest.raises(Exception):
```

Comment:
can we make this more specific and check it raises a `ValueError` rather than a generic Exception?